api/testing/restapi_test.go

```diff
@@ -0,0 +1,686 @@
+package testing
```

Comment:
This file should be under `api/client/volume` since it is testing that package.

Reply:
@lpabon this test suite tests both client as well as server code. All the tests will invoke both http client and server with the driver as MockDriver. So does it not make sense for the tests to be under `apt/test`.